From 8f3fd1a84f9db6cea47e6badd45e0d5023011ff3 Mon Sep 17 00:00:00 2001
From: Paul Keith <javelinanddart@gmail.com>
Date: Fri, 6 May 2022 10:41:52 +0530
Subject: [PATCH 1/3] libstagefright: omx: Add support for loading prebuilt ddp
 and ac4 decoder lib

Change-Id: I3b85c15889c9e3aef152e726a83936234103254e

OMX: Remove support for prebuilt ac4 decoder

Requires proprietary Dolby changes in libstagefright.

This partially reverts commit 01090fa125c2993654e18e024ddf7cef87bca30e.

Change-Id: Ifdf36ad253155e77ba14d0624612bf6be6711ca6
---
 media/libstagefright/omx/SoftOMXPlugin.cpp | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/media/libstagefright/omx/SoftOMXPlugin.cpp b/media/libstagefright/omx/SoftOMXPlugin.cpp
index 8c186c90f8..6258ee08d7 100644
--- a/media/libstagefright/omx/SoftOMXPlugin.cpp
+++ b/media/libstagefright/omx/SoftOMXPlugin.cpp
@@ -61,6 +61,9 @@ static const struct {
     { "OMX.google.flac.decoder", "flacdec", "audio_decoder.flac" },
     { "OMX.google.flac.encoder", "flacenc", "audio_encoder.flac" },
     { "OMX.google.gsm.decoder", "gsmdec", "audio_decoder.gsm" },
+    { "OMX.dolby.ac3.decoder", "ddpdec", "audio_decoder.ac3" },
+    { "OMX.dolby.eac3-joc.decoder", "ddpdec", "audio_decoder.eac3_joc" },
+    { "OMX.dolby.eac3.decoder", "ddpdec", "audio_decoder.eac3" },
 };
 
 static const size_t kNumComponents =
-- 
2.47.1

